packages/sanity/src/tasks/src/tasks/components/sidebar/TasksSidebarHeader.tsx

```diff
@@ -74,6 +75,14 @@ export function TasksSidebarHeader(props: TasksSidebarHeaderProps) {
             onClick={handleTaskCreate}
             mode="bleed"
             text={t('buttons.new.text')}
+            disabled={mode === 'upsell'}
+            tooltipProps={
```

Comment:
should we actually have a tooltip in default mode?

Reply:
When the mode is `default` it won't render the tooltip because this prop will be undefined.
The `button` checks if this prop `tooltipProps` is defined or not, it will only add the tooltip if it's defined.

For default mode the text in the button explains what it will do, so I think we don't need a tooltip for that mode.